mpas_analysis/ocean/meridional_heat_transport.py

```diff
@@ -0,0 +1,218 @@
+"""
+Plot meridional heat transport from the analysis member output.
+
+Authors
+-------
+Mark Petersen, Milena Veneziani, Phillip Wolfram, Xylar Asay-Davis
+
+Last Modified
+-------------
+04/11/2017
+"""
+
+import xarray as xr
+import numpy as np
+import netCDF4
+import os
+import warnings
+
+from ..shared.constants.constants import m3ps_to_Sv, rad_to_deg
+from ..shared.plot.plotting import plot_vertical_section,\
+    timeseries_analysis_plot, setup_colormap
+from ..shared.io.utility import build_config_full_path
+from ..shared.generalized_reader.generalized_reader \
+    import open_multifile_dataset
+from ..shared.timekeeping.utility import get_simulation_start_time, \
+    days_to_datetime
+from ..shared.analysis_task import setup_task
+from ..shared.climatology import climatology
+from ..shared.constants import constants
+
+
+def meridional_heat_transport(config):  # {{{
+    """
+    Process MHT analysis member data if available.
+    Plots MHT as:
+       1D function of latitude
+       2D function of latitude and depth
+
+    Parameters
+    ----------
+    config : instance of MpasAnalysisConfigParser
+        configuration options used to customize the analysis task
+
+    Authors
+    -------
+    Mark Petersen, Milena Veneziani, Phillip Wolfram, Xylar Asay-Davis
+
+    Last Modified
+    -------------
+    04/11/2017
+    """
+
+    # **** Initial settings ****
+
+    # perform common setup for the task
+    namelist, runStreams, historyStreams, calendar, streamMap, \
+        variableMap, plotsDirectory = setup_task(config, componentName='ocean')
+
+    try:
+        timeSeriesStatsMonthlyAnalysisMemberFlag = namelist.get(
+            'config_am_timeseriesstatsmonthly_enable')
```

Comment:
This will need to be updated to use the namelistMap, following the approach used in MOC.  The call to `setup_task` will be missing the `namelistMap` return value after you rebase onto `develop`.